migrations/0017_create_tickets.up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS tickets (
+    id INTEGER PRIMARY KEY NOT NULL,
+
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    quantity INTEGER NOT NULL,
+    max INTEGER NOT NULL,
+    kind TEXT NOT NULL,
+    sort INTEGER NOT NULL,
+
+    -- kind = 'fixed'
+    price INTEGER,
+    -- kind = 'variable'
+    price_min INTEGER,
```

Comment:
nit: "price" -> "contribution" for consistency with user-facing language

Reply:
it felt a bit awkward: paying $20 to go to an event is "a contribution", but you wouldn't call $20 "the contribution" of the event

Reply:
maybe `dollars`, `dollars_min`, etc?

Reply:
at the end of the day it's still "reserving a spot for a price", even if we gratefully refer to that as "a contribution" rather than "paying a price" in user facing copy. code wise we should prefer clarity

Reply:
ah `required_contribution` is great! going with that